storage/txcache/instrumentation.go

```diff
@@ -0,0 +1,55 @@
+package txcache
+
+import (
+	"github.com/ElrondNetwork/elrond-go/logger"
+)
+
+var mainLog = logger.GetOrCreate("txcache")
+
+func (cache *TxCache) onEvictionStarted() {
+	info("TxCache.onEvictionStarted()")
+	cache.displayState()
+}
+
+func (cache *TxCache) onEvictionEnded() {
+	info("TxCache.onEvictionEnded()")
+	cache.evictionJournal.display()
+	cache.displayState()
+}
+
+func (cache *TxCache) displayState() {
+	txListBySenderMap := cache.txListBySender.backingMap
+	info("TxCache:", "numBytes", cache.NumBytes(), "txs", cache.CountTx(), "senders", cache.CountSenders())
+	info("TxCache.txListBySender.totalCounts:", "chunks", txListBySenderMap.Count(), "iscoreChunks", txListBySenderMap.CountSorted())
+	info("TxCache.txListBySender.counts:", "chunks", txListBySenderMap.ChunksCounts(), "scoreChunks", txListBySenderMap.ScoreChunksCounts())
+}
+
+func (cache *TxCache) onRemoveTxInconsistency(txHash []byte) {
+	// This should never happen (eviction should never cause this kind of inconsistency between the two internal maps)
+	mainLog.Error("TxCache.onRemoveTxInconsistency(): detected maps sync inconsistency", "tx", txHash)
```

Comment:
in the project logs are written with log.Error, log.Warn - the same format should be kept.

Reply:
Renamed `mainLog` to `log`.